docs/knowledge_base/tips/Big data models vs. Computer memory.md

```diff
@@ -0,0 +1,462 @@
+2. Intermediate Data Nodes: We’ll need to add four more Data Nodes: “*scored_df”*, “*segmented_customer_df*”, “*segment_result*”, “*high_value_summary_df*”. With Taipy’s intelligent design, you don’t need to configure anything for these intermediate data nodes; the system handles them smartly.
+3. **Intermediate Data Nodes with Defaults:** We finally define four more intermediate Data Nodes with the “*default_data*” property set to the following:
+- payment_threshold: “1000:int”
+
+![Data Nodes](data_nodes.png)
+
+
+- score_threshold: “1.5:float”
+- metric: “mean”
+- summary_statistic_type: “median”
+
+<hr/>
+
+### Tasks
+Clicking on the Add Task button, you can configure a new Task. Add four Tasks, then **link each Task to the appropriate function** under the Details tab. Taipy Studio will scan through your project folder and provide a categorized list of functions to choose from, sorted by the Python file.
+
+**Task 1** (```preprocess_and_score```): In Taipy studio, you’d click the Task icon to add a new Task. You’d specify the input as “*path_to_data*” and the output as “*scored_df*”. Then, under the Details tab, you’d link this Task to the algos.```algo.preprocess_and_score``` function.
+
+![preprocess](preprocess.png)
+
+**Task 2** (```featurization_and_segmentation```): Similar to Task 1, you’d specify the inputs (”*scored_df*”, ”*payment_threshold*”, ”*score_threshold*”) and the output (”*segmented_customer_df*”). Link this Task to the ```algos.algo.featurization_and_segmentation``` function.
+
+![Featurization](featurization.png)
+
+**Task 3** (```segment_analysis```): Inputs would be “*segmented_customer_df*” and “*metric*”, and the output would be “*segment_result*”. Link to the ```algos.algo.segment_analysis``` function.
+
+![Segment Analysis](segment_analysis.png)
+
+**Task 4** (```high_value_cust_summary_statistic```): Inputs include “*segment_result*”, “*segmented_customer_df*”, and “*summary_statistic_type*”. The output is “*high_value_summary_df*”. Link to the algos.algo.```high_value_cust_summary_statistics``` function.
+
+![High Value](high_value.png)
+
+### Conclusion
+
+Taipy offers an i**ntelligent way to build and manage data pipelines**. The skippable feature in particular, makes it a powerful tool for optimizing computational resources and time, particularly beneficial in scenarios involving large datasets. While **Dask provides the raw power for data manipulation, Taipy adds a layer of intelligence**, making your pipeline not just robust but also smart.
+
+***Additional Resources***
+
+For the complete code and TOML configuration, you can visit this [GitHub repository](https://github.com/Avaiga/demo_dask_customer_analysis). To dive deeper into Taipy, here’s the [official documentation](https://docs.taipy.io/en/latest/).

```

Comment:
This article is deep in the doc site. It's strange to say that to get more info you need to dive into the home page of the article.